ida-ws/src/main/java/upb/ida/logging/AspectLogger.java

```diff
@@ -0,0 +1,143 @@
+package upb.ida.logging;
+
+/**
+ * Logs Creation - Using Spring AOP to provide universal logging of all the
+ * exceptions and method calls.
+ * 
+ * @author Ayaz
+ *
+ */
+
+import java.io.File;
+
+import javax.servlet.http.HttpServletRequest;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import upb.ida.bean.ResponseBean;
+import upb.ida.constant.IDALiteral;
+import org.apache.log4j.BasicConfigurator;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+
+
+@Component
+@Aspect
+public class AspectLogger {
+
+	@Autowired(required=true)
+	private HttpServletRequest request;
+	@Autowired
+	private ResponseBean response;
+	private Logger requestReponseLogger;
+	private Logger exceptionLogger;
+	private Logger datasetResponseLogger;
+	
+    /**
+     * Constructor of AspectLogger class
+     */
+	AspectLogger()
+	{
+		BasicConfigurator.configure();
+		String log4jConfigFile = System.getProperty("user.dir") + File.separator + "log4j.properties";
```

Comment:
After moving the log4j file to src/main/resources, use FilteUtil.fetchSysFilePath instead.

(Ref: https://github.com/dice-group/ida/blob/master/ida-ws/src/main/java/upb/ida/util/FileUtil.java#L199)